### PR TITLE
Add Splash Screen

### LINKED ISSUE #545: semux-gui started slowly
Hi there,

#### System information

Semux version: `v1.0.0`

OS & Version: `macOS 10.12.6 `

Java Version:`java version "1.8.0_144"
Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed mode)`

#### Expected behaviour

semux-gui started quickly

#### Actual behaviour

semux-gui started slowly

#### Steps to reproduce the behaviour

run  `./semux-gui.sh` shell,I need to wait 5 seconds or 10 seconds.

RC4 and RC5 Version started quickly,please improve performance.

### PR DESCRIPTION
**Pull request by cryptokat**
fix #545 

The source file of splash screen design is stored as a [Krita](https://krita.org/en/) document: `misc/design/splash.kra`

### CODE REVIEW

**Comment by semuxdev** [comment]
Can we start splash screen after the selection of coinbase account?

screenshot from 2018-03-25 22-34-10

**Comment by cryptokat** [comment]
@semuxdev fixed by https://github.com/semuxproject/semux/pull/744/commits/a6f853e7ef48d5fb1be657a1465e9d66c8a68106

File: src/main/java/org/semux/gui/SemuxGui.java [hunk 1]
```diff
@@ -70,6 +70,7 @@
     protected AddressBookDialog addressBookDialog;
 
     protected boolean isRunning;
+    private SplashScreen splashScreen;
```
Comment (orogvany) [inline src/main/java/org/semux/gui/SemuxGui.java]: maybe scope to protected to match rest in this class, or move it to separate section. /pedant off

File: src/main/java/org/semux/gui/SplashScreen.java [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This splash screen fills the empty between startup dialog and
+ * {@link MainFrame}.
+ */
+public class SplashScreen extends JFrame implements SemuxEventListener {
+
+    private static final long serialVersionUID = 5729665012428583185L;
```
Comment (orogvany) [inline src/main/java/org/semux/gui/SplashScreen.java]: In other classes, we use 
        private static final long serialVersionUID = 1L;
extensively.

File: src/main/java/org/semux/gui/SplashScreen.java [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    }
+
+    private void destroySplash() {
+        setVisible(false);
+        dispose();
+    }
+
+    private class ContentPane extends JPanel {
+
+        private static final long serialVersionUID = -5929022627261315407L;
```
Comment (orogvany) [inline src/main/java/org/semux/gui/SplashScreen.java]: same here, and below.

**Pull request merged**